src/DynamoCoreWpf/Views/PackageManager/Controls/PackageManagerPackagesControl.xaml.cs

```diff
@@ -187,6 +186,37 @@ private void DropDownInstallButton_OnClick(object sender, RoutedEventArgs e)
 
             // Open the context menu when the left mouse button is pressed
             contextMenu.IsOpen = true;            
-        }   
+        }
+
+        // Attach the 'context menu' button to the parent button
+        private void DropDownInstallButton_Loaded(object sender, RoutedEventArgs e)
+        {
+            var installButton = sender as Button;
+            if (installButton != null)
+            {
+                var dropDownInstallButton = installButton.Template.FindName("dropDownInstallButton", installButton) as Button;
+                if (dropDownInstallButton != null)
+                {
+                    dropDownInstallButton.Click -= DropDownInstallButton_OnClick;
```

Comment:
?? It appears you unsub and resub the same event to the same method?

Reply:
Yes, because the 'Loaded' somehow gets called a few times which leads to multiple subscriptions. This way I make sure I only subscribe once.

Reply:
okay, hah, if you look this up, the consensus is yes it works, but it's an anti pattern, of course I can't think of anything better that won't require a big refactoring - so let's proceed 😉

Reply:
I was looking for a way to check if the Click event is already subscribed, and only then to unsubscribe to make it a bit more civil, but there isn't such a thing it seems .. (this is what I found https://stackoverflow.com/a/2697698/4749826)